packages/apollo-forest-run/src/diff/diffErrorKind.ts

```diff
@@ -0,0 +1,3 @@
+export const MissingModelValue = 0,
```

Comment:
question: isnt fallback for enum object instead of desacralizing into single properties?

Reply:
Literal object cannot be inlined because `const` keyword doesn't prevent object property mutations. For now, only scalar values with `const` keyword could be safely inlined by minifiers. 

`export * as Foo from "foo"` is not the same as an object with the same set of properties - bundlers like Webpack know that they can treat this object as immutable collection of ES exports and so can inline it too.